sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -843,6 +847,9 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     test("2015-07-24 00:00:00", null, null)
     test(null, null, null)
     // Test escaping of timezone
-    GenerateUnsafeProjection.generate(FromUTCTimestamp(Literal(0), Literal("\"quote")) :: Nil)
+    val msg = intercept[java.time.DateTimeException] {
+      GenerateUnsafeProjection.generate(FromUTCTimestamp(Literal(0), Literal("\"quote")) :: Nil)
+    }.getMessage
+    assert(msg == "Invalid ID for region-based ZoneId, invalid format: \"quote")
```

Comment:
This is fine, but you could weaken this to just check that it includes `\"quote` to not rely on the specific error message

Reply:
done